.gitignore

```diff
@@ -9,3 +9,6 @@ dist
 jscoverage.json
 tags
 .cake_task_cache
+
+# Visual Studio Code
+.vscode/
```

Comment:
This shouldn't go in here -- non-repo specific ignores should be handled by the user's global git ignore. No need to repush; I can remove this when I merge.

(The other editor-specific entries in Vimium's .gitignore are there by historical accident and are due to be removed).